src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: 600;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form-big {
+  margin: 20px 0;
+  position: relative;
+}
+
+.form-small {
+  position: relative;
+}
+
+.form-big::before {
+  content: "";
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  align-items: center;
+  bottom: 26px;
+  width: 19px;
+  height: 19px;
+  background: url("images/Search.svg") no-repeat center / contain;
+}
+
+.form-small::before {
+  content: "";
+  position: absolute;
+  left: 13px;
+  top: 15px;
+  bottom: 16px;
+  width: 11px;
+  height: 11px;
+  background: url("images/Search.svg") no-repeat center / contain;
```

Comment:
You have duplicated rules for two selectors (background, position, content)
Use `,` selector for combining rules